lib/site-utils.js

```diff
@@ -83,9 +86,12 @@ exports.processPrice = function processPrice(priceStringInput) {
     logger.log('found yen in price, converting to number...');
     price = accounting.unformat(priceString);
   } else if (_.includes(priceString, GBP_SYMBOL_PREFIX) ||
-    _.includes(priceString, GBP_SYMBOL_PREFIX)) {
+    _.includes(priceString, GBP_TEXT_PREFIX)) {
```

Comment:
Initially, it was an redundant check, so I just change it to how it should be! :smile: